holoviews/plotting/bokeh/element.py

```diff
@@ -355,12 +356,24 @@ def _title_properties(self, key, plot, element):
         else:
             title = ''
 
-        opts = dict(text=title, text_color='black')
+        theme = self.renderer.theme
+        if theme is not None:
+            if theme in built_in_themes:  # built-in-theme name
+                opts = built_in_themes[theme]._json['attrs']['Title']
+            else:  # theme object
+                opts = theme._json['attrs']['Title']
+            opts['text'] = title
+        else:
+            opts = dict(text=title, text_color='black')
```

Comment:
I feel like there's a way to simplify this logic, but my brain isn't working right now...

Reply:
Since the default text_color is black anyway I think we can simply remove this bit entirely.